app/services/provider_interface/sort_application_choices.rb

```diff
@@ -18,13 +18,12 @@ def self.for_task_view(application_choices)
           WHEN #{inactive} THEN 1
           WHEN #{awaiting_provider_decision} THEN 2
           WHEN #{deferred_offers_pending_reconfirmation} THEN 3
-          WHEN #{give_feedback_for_rbd} THEN 4
-          WHEN #{interviewing} THEN 5
-          WHEN #{pending_conditions_previous_cycle} THEN 6
-          WHEN #{waiting_on_candidate} THEN 7
-          WHEN #{pending_conditions_current_cycle} THEN 8
-          WHEN #{successful_candidates} THEN 9
-          WHEN #{deferred_offers_current_cycle} THEN 10
+          WHEN #{interviewing} THEN 4
+          WHEN #{pending_conditions_previous_cycle} THEN 5
+          WHEN #{waiting_on_candidate} THEN 6
+          WHEN #{pending_conditions_current_cycle} THEN 7
+          WHEN #{successful_candidates} THEN 8
+          WHEN #{deferred_offers_current_cycle} THEN 19
```

Comment:
```suggestion
          WHEN #{deferred_offers_current_cycle} THEN 9
```

I know this is draft, but I saw it when looking at error in your build.

Reply:
Haha thanks. Forgot what comes after 8 😅